device-types/Cisco/N9K-C93360YC-FX2.yml

```diff
@@ -0,0 +1,252 @@
+  - name: Ethernet1/53
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/54
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/55
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/56
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/57
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/58
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/59
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/60
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/61
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/62
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/63
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/64
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/65
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/66
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/67
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/68
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/69
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/70
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/71
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/72
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/73
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/74
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/75
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/76
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/77
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/78
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/79
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/80
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/81
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/82
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/83
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/84
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/85
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/86
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/87
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/88
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/89
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/90
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/91
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/92
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/93
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/94
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/95
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/96
+    type: 25gbase-x-sfp28
+  - name: Ethernet1/97
+    type: 100gbase-x-qsfp28
+  - name: Ethernet1/98
+    type: 100gbase-x-qsfp28
+  - name: Ethernet1/99
+    type: 100gbase-x-qsfp28
+  - name: Ethernet1/100
+    type: 100gbase-x-qsfp28
+  - name: Ethernet1/101
+    type: 100gbase-x-qsfp28
+  - name: Ethernet1/102
+    type: 100gbase-x-qsfp28
+  - name: Ethernet1/103
+    type: 100gbase-x-qsfp28
+  - name: Ethernet1/104
+    type: 100gbase-x-qsfp28
+  - name: Ethernet1/105
+    type: 100gbase-x-qsfp28
+  - name: Ethernet1/106
+    type: 100gbase-x-qsfp28
+  - name: Ethernet1/107
+    type: 100gbase-x-qsfp28
+  - name: Ethernet1/108
+    type: 100gbase-x-qsfp28
+  - name: mgmt0
+    type: 1000base-t
+    mgmt_only: true
+  - name: mgmt1
+    type: 25gbase-x-sfp28
```

Comment:
I believe this is a SFP+ port.

Reply:
The documentation on the cisco site has it listed as a sfp port not sfp plus so that is what I listed it as.  https://www.cisco.com/c/en/us/td/docs/switches/datacenter/nexus9000/hw/n93360yc-fx2-hig/b_c93360yc-fx2-nxos-mode-hardware-installation-guide/b_c93360yc-fx2-nxos-mode-hardware-installation-guide_chapter_01.html

Reply:
It's certainly not SFP28, though. :-)

Reply:
I updated it to '1000base-x-sfp'